inputmodule-control/Makefile.toml

```
@@ -0,0 +1,8 @@
+extend = "../Makefile.toml"
+
+[env]
+TARGET_TRIPLE = "x86_64-unknown-linux-gnu"
```

```suggestion
TARGET_TRIPLE = "${CARGO_MAKE_RUST_TARGET_TRIPLE}"
```

Awesome, this works!